ansible/influxdb-meta/tests/vagrant.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+  pre_tasks:
+    - name: "Populate hosts file"
```

Comment:
What happened on the CentOS hosts ? It seems really strange that would not work on both families?

As this is for testing only, I think we should just keep the logic in the `vagrant.yml`, people really shouldn't be using our role to setup their hosts file, ideally they should already have that done with another module, and most ideally, have DNS running at their site.  Hence why I pulled this stuff out of our role.